test/test-helpers.ts

```diff
@@ -172,6 +174,8 @@ export function printToDoc(content: string, options: any = {}) {
 }
 
 export function debug(content: string, options: any = {}) {
+  const ast = parse(content, plugin.parsers, options);
+  const processedAST = preprocess(ast, options);
```

Comment:
added those two because they are handy when prefixing an integration test with `debug`.

It'll drop you somewhere where you can inspect the trees.

![](https://screenshot.click/02-13-y4jwr-33pdh.png)